pyClient/commands/utils.py

```diff
@@ -17,6 +17,7 @@
 from os.path import exists, join
 from solcx import compile_files  # type: ignore
 from typing import Dict, Tuple, Optional, Any
+from web3 import Web3
```

Comment:
```suggestion
from web3 import Web3  # type: ignore
```
I think this line is interferring with the type checker.  Disabling type checking should fix the travis builds.

Reply:
Thanks!